### ISSUE TITLE
Shrine: Primary color does not have enough contrast ratio

### ISSUE THREAD

**Issue by pennzht** [material-components/material-components-flutter-gallery#181]
## Issue
* On Login screen, text buttons are using a primary color that does not have a sufficient contrast ratio.
* This also affects text field's active color.

## Acceptance Criteria
* Shrine text button has enough contrast ratio.
* Ask designer about what to do.

## Details
* Use `442C2E` (or shrine's `onSurface`) for text buttons and textfield cursors.
